src/views/protected/Notification.vue

```diff
@@ -28,9 +29,32 @@
 
 <script>
 import Notification from "@/components/notifications/Notification.vue";
+import { notificationService } from "@/services";
 
 export default {
   name: "Notification",
   components: { NotifyComponent: Notification },
+  data(){
+    return {
+     notifications: null
+    }
+  },
+  methods: {
+    getNotification() {
+      notificationService.getNotification()
+        .then((notifications) => {
+          console.log(notifications);
+          this.notifications = notifications;
+        })
+        .catch((error) => {
+          console.log(error);
```

Comment:
You can create your own error in `messagingConfig` and throw that directly. Error message should be like: `'Failed to retrieve {0}. Please try again'`